.github/workflows/tests.yml

```diff
@@ -1,6 +1,12 @@
 ---
 name: Tests
-on: [push, pull_request]
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
```

Comment:
Would just `on: [push]` do? (Such still shows up when you open a PR I think)

Reply:
`on: [push]` probably doesn't work if the source is in a fork.  The GitHub Action still runs, but only in the fork and only if the haven't been disabled there.